bindings/python/Planners/tests/test_time_verying_dcm_planner.py

```diff
@@ -7,7 +7,7 @@
 
 def test_dcm_planner_state():
 
-    state = blf.DCMPlannerState()
+    state = blf.planners.DCMPlannerState()
```

Comment:
Not related to this line, but I noticed that the file name has a typo (my fault).

Reply:
I can fix that :)